### ISSUE TITLE
Gold Price visual tweaks

### ISSUE THREAD

**Issue by saraolson** [HabitRPG/habitica-android#843]
1. Prices on gold purchasable item cards are too light, making them hard to read. Same with the gold prices on buy buttons. This affects shop items and reward screen items

Should be: #EE9109 (Yellow 5)

2. On custom rewards: the gold icon appears smaller than the gold icon used in other places. It should actually be slightly larger than the gold icon used in the header and on cards
